### PR TITLE
Robots.txt with multilang/multishop

### LINKED ISSUE #12032: robot module disallow
Prestashop 1.7.5
Linux #1 SMP Debian 4.9.110-3+deb9u2 (2018-08-13) x86_64
Apache
Version PHP: 7.2.13
default prestashop robots file disallow /modules, so search engine look at module folder.
but my psfeeder works only on module folder--> https://www.100asa.it/it/module/ps_feeder/rss?id_category=0&orderby=date_upd&orderway=desc

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.5.x
| Description?  | Add the correct path to Disallow directories when using a multishop or a multilang instance.
| Type?         | bug fix
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #12032
| How to test?  | Create a multilang instance and make sure you have Disallow with all your languages. Same with a multishop instance and physical path.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12106)

### CODE REVIEW

File: classes/Tools.php [hunk 1]
```diff
@@ -2447,13 +2459,12 @@ public static function generateHtaccess($path = null, $rewrite_settings = null,
...
                 fwrite($write_fd, PHP_EOL . PHP_EOL . '#Domain: ' . $domain . PHP_EOL);
                 if (Shop::isFeatureActive()) {
-                    fwrite($write_fd, 'RewriteCond %{HTTP_HOST} ^' . $domain . '$' . "\n");
+                    fwrite($write_fd, 'RewriteCond %{HTTP_HOST} ^' . $domain . '$' . PHP_EOL);
                 }
-                fwrite($write_fd, 'RewriteRule . - [E=REWRITEBASE:' . $uri['physical'] . ']' . "\n");
+                fwrite($write_fd, 'RewriteRule . - [E=REWRITEBASE:' . $uri['physical'] . ']' . PHP_EOL);
 
                 // Webservice
                 fwrite($write_fd, 'RewriteRule ^api$ api/ [L]' . "\n\n");
```
Comment (MathiasReker) [inline classes/Tools.php:2478]: PHP_EOL

Comment (PierreRambaud) [reply classes/Tools.php:2478]: I will not change every \n in this class :P 
It was just a fast replace for my needs =)

Comment (MathiasReker) [reply classes/Tools.php:2478]: Okay :)

File: classes/Tools.php [hunk 2]
```diff
@@ -2321,6 +2321,44 @@ public static function getMediaServer($filename)
         return Tools::usingSecureMode() ? Tools::getShopDomainSsl() : Tools::getShopDomain();
     }
 
+    /**
+     * Get domains information
```
Comment (Quetzacoalt91) [inline classes/Tools.php]: Would it be possible to have more details on the array content?

Comment (PierreRambaud) [reply classes/Tools.php]: Is it better? I don't know what to say about this method :/

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by matks** [comment]
Thanks @PierreRambaud

**Pull request merged**